packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -68,7 +67,7 @@ export const EventMeta = () => {
           <EventTitle className="my-2">{event?.title}</EventTitle>
           {event.description && (
             <EventMetaBlock contentClassName="mb-8 break-words max-w-full max-h-[180px] scroll-bar pr-4">
-              <div dangerouslySetInnerHTML={{ __html: markdownToSafeHTML(event.description) }} />
+              <div dangerouslySetInnerHTML={{ __html: event.description }} />
```

Comment:
getPublicEvent already ensures that `event.description` is gone through `markdownToSafeHTML`